metron-analytics/metron-profiler-storm/pom.xml

```diff
@@ -301,14 +301,7 @@
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
-            <version>${global_mockito_version}</version>
             <scope>test</scope>
```

Comment:
Sure about removing this exclusion? At one point there was some drama between versions of mockito and what it pulled in for hamcrest.

Reply:
I believe mockito dropped the explicit pulling in of hamcrest in the version I had to bump to. I'm not sure what the original drama was though.